src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
```

Comment:
The import statement for `core.basesyntax.db.FruitTransaction` is not used in this interface. If this import is not needed, it should be removed to keep the code clean and maintainable.